test/controllers/PaymentHistoryControllerSpec.scala

```diff
@@ -55,6 +55,7 @@ class PaymentHistoryControllerSpec extends MockAuthenticationPredicate
       MockAuthenticationPredicate,
       app.injector.instanceOf[NinoPredicate],
       MockIncomeSourceDetailsPredicate,
+      mockAuditingService,
```

Comment:
Why don't we verify that `AuditingService` was called with expected values in controllers tests?

Reply:
It's down to the effort required at this level I believe. To verify the audit in the controller, we have to verify the entire MtdItUser matches what we expect which contains lots of premade test data for most scenarios already. This only verifies that the auditing service gets the information we expect it to get, not that the resulting call was made or whether it contained the correct information.

In integration tests, in theory it becomes much simpler to match up the few fields you need and it also checks calls made + content.